packages/viz/src/lib/sources/DOSource.ts

```diff
@@ -13,7 +13,32 @@ interface Variable {
   name: string;
   slug: string;
   starred: boolean;
-  summary_json: any;
+  summary_json: {
```

Comment:
We should follow a convention for all of our interfaces on camelCase vs snake_case